manual_jb/content/analyses/tornadoepn_dmg.md

```diff
@@ -28,8 +28,7 @@ key name | type | name | description
 
 key name | type | name | description
 --- | --- | --- | ---
-`epn_node` <sup>*</sup> | `incore:epnNodeVer1` | EPN node | A dataset with nodes of the electric power network.
-`epn_link` <sup>*</sup> | `incore:epnLinkeVer1` | EPN link | A dataset with links of the electric power network.
+`epn_network_dataset` <sup>*</sup> | `incore:epnNodeVer1` | EPN network dataset | A network dataset of the electric power network.
```

Comment:
I believe this should be epn_network for the keyname and the type is incore:epnNetwork

Reply:
updated